src/GitHub.TeamFoundation.14/Services/VSGitExt.cs

```diff
@@ -118,6 +127,7 @@ private set
             {
                 if (value != activeRepositories)
                 {
+                    log.Debug("ActiveRepositories changed to {Repositories}", value.Select(x => x.CloneUrl));
```

Comment:
I'm curious what happens when `value` is `null`. I know `Select` is an extension method. Does it simply return `null`?

Reply:
It looks like it would throw an `ArgumentNullException`.

Reply:
Ah yeah, I was thinking it wouldn't be null but it can when `gitService` is null. Fixed.